src/containers/pages/OrganizationViews/AssignPractitioners/index.tsx

```diff
@@ -112,9 +113,6 @@ const AssignPractitioner = (props: PropsTypes) => {
     chosenOptions: ValueType<SelectOption>,
     { action, removedValue, option }: any
   ) => {
-    if (!chosenOptions) {
-      return;
-    }
```

Comment:
@p-netm is this what was blocking the component from deselecting the final option?

Reply:
@cKellyDesign not exactly the final option but the only the first selected option.
Steps to reproduce:
- open an organizations without assigned practitioners
- select one practitioner from the select dropDwon
- try to remove it